tests/testutils/goroutineassistant.go

```diff
@@ -18,23 +18,25 @@ import (
 	"fmt"
 	"sync"
 	"testing"
+
+	"github.com/coreos/rkt/Godeps/_workspace/src/github.com/steveeJ/gexpect"
 )
 
 type GoroutineAssistant struct {
-	s  chan string
+	s  chan error
 	wg sync.WaitGroup
 	t  *testing.T
 }
 
 func NewGoroutineAssistant(t *testing.T) *GoroutineAssistant {
 	return &GoroutineAssistant{
-		s: make(chan string),
+		s: make(chan error),
 		t: t,
 	}
 }
 
 func (a *GoroutineAssistant) Fatalf(s string, args ...interface{}) {
-	a.s <- fmt.Sprintf(s, args...)
+	a.s <- fmt.Errorf(s, args...)
```

Comment:
I would just make it fixed at `10` and that's it. For now `go` does not allow to grow channel's buffer. For now we don't use too many goroutines on single assistant (2 at most), so it should be enough in the forseeable future.